openmdao/docs/features/debugging/om_command.rst

```diff
@@ -215,6 +215,6 @@ Using Commands under MPI
 In general, usage of openmdao subcommands under MPI is the same as usual, except the command will
 be preceded by `mpirun -n <num_procs>`.  For example:
 
-.. embed-shell-cmd::
-    :cmd: mpirun -n 2 openmdao summary multipoint_beam_opt.py
-    :dir: ../test_suite/scripts
+.. .. embed-shell-cmd::
+..     :cmd: mpirun -n 2 openmdao summary multipoint_beam_opt.py
+..     :dir: ../test_suite/scripts
```

Comment:
so this is broken?  noticed you had un-commented in previous commit. just making sure this is intentional

Reply:
@jubs, have you seen this (above) question?

Reply:
errr @JustinSGray

Reply:
(If it is broken and stays commented out, should probably remove "For example:" above it, so it doesn't say, "For example: NOTHING" )